pkg/parser/parser.go

```diff
@@ -199,6 +207,9 @@ func (p *Parser) parseStatement(scope *scope) Node {
 		if p.isFuncCall(p.cur) {
 			return p.parseFunCallStatement(scope)
 		}
+		if p.peek.Type == lexer.LBRACKET || p.peek.Type == lexer.DOT {
```

Comment:
I had to move this down, otherwise `len [1]` was interpreted as ASSIGNMENT in errored. I just moved the p.isFuncCall line before assignment I got less useful error messages for `print = 5` etc.

Reply:
I did not. Will move it back up.